sound/soc/sof/intel/hda-loader.c

```diff
@@ -165,19 +153,15 @@ static int cl_dsp_init(struct snd_sof_dev *sdev, const void *fwdata,
 					HDA_DSP_SRAM_REG_ROM_STATUS,
 					HDA_DSP_ROM_STS_MASK, HDA_DSP_ROM_INIT,
 					HDA_ROM_INIT_TIMEOUT);
-	if (ret >= 0)
+	if (!ret)
 		goto out;
```

Comment:
nitpicking: a "return 0" would be sufficient and IMHO a bit clearer here. You could remove the "out" label then too.